react-app/src/components/atoms/Alert/AlertComponent.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { AlertPopup } from './alert';
+
+const AlertComponent = () => {
+  let type = 'success';
+  const visible = useSelector(({ modals: { alert: visibility } }) => visibility);
```

Comment:
Consider using [reselect library](https://www.npmjs.com/package/reselect) and reusable selectors in general. That would make your code more flexible and your app more performant.